test/framework/src/main/java/org/elasticsearch/AnalysisFactoryTestCase.java

```diff
@@ -228,6 +228,10 @@
         .put("fingerprint",               Void.class)
         // for tee-sinks
         .put("daterecognizer",            Void.class)
+        .put("minhash",                   Void.class)
```

Comment:
cool - can you maybe send a mail to other devs that these are available and we should expose them?